### DISCUSSION TITLE
Is possible to access old versions of module

### DISCUSSION THREAD

**Discussion by ufoloko** [openstore-ecommerce/OpenStore#300] [General]
I'm looking for source code for version 2.3.1

Is possible to access that version ?

**Comment by leedavi** [comment]
v2.... wow that is old.  

It was archived on codeplex.   It's a near rewrite since v2.  v3 rewrote the architecture,   At he moment Codeplex archive is not rsponing to me.  But search for NBrightBuy on there.

**Comment by leedavi** [comment]
https://codeplexarchive.org/codeplex/project/nbrightbuy
